3-Network-Configuration/3.5-Firewall-Configuration.bats

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env bats
+
+load IPv6-helper
+
+# 3.5.1 Configure UncomplicatedFirewall
+
+@test "3.5.1.1 Ensure ufw is installed (Automated)" {
+    [[ $(dpkg -s ufw | grep 'Status: install') == "Status: install ok installed" ]]
+}
+
+@test "3.5.1.2 Ensure iptables-persistent is not installed with ufw (Automated)" {
+    run dpkg-query -s iptables-persistent
+    [ $status -eq 1 ]
+    [[ "$output" == *"package 'iptables-persistent' is not installed and no information is available"* ]]
+}
+
+@test "3.5.1.3 Ensure ufw service is enabled (Automated)" {
+    [[ $(systemctl is-enabled ufw) == "enabled" ]]
+    [[ $(ufw status | grep Status) == "Status: active" ]]
+}
+
+@test "3.5.1.4 Ensure ufw loopback traffic is configured (Automated)" {
+    run bash -c "ufw status verbose"
+    [ "$status" -eq 0 ]
+    [[ "$output" == *"Anywhere on lo"*"ALLOW IN"*"Anywhere"* ]]
```

Comment:
This statement doesn't make sure that all the strings are in the same line. Maybe use the `$lines` array: https://bats-core.readthedocs.io/en/latest/writing-tests.html